ci-operator/config/maistra/istio-operator/maistra-istio-operator-maistra-2.2.yaml

```diff
@@ -32,6 +32,29 @@ tests:
     XDG_CACHE_HOME="/tmp/cache"
   container:
     from: src
+- as: update-charts-in-2-4
+  commands: |
+    # load test-infra automator.sh script
+    git clone https://github.com/maistra/test-infra.git
+    cd test-infra
+    export BUILD_WITH_CONTAINER="0"
+    export GOFLAGS="-mod=vendor"
+    export XDG_CACHE_HOME="/tmp/cache"
+    export GOCACHE="/tmp/cache"
+    export GOMODCACHE="/tmp/cache"
+    ./tools/automator.sh \
+      -o maistra \
+      -r istio-operator \
+      -b maistra-2.4 \
+      -f /creds-github/token \
+      '-c make clean update-charts-2.3' \
```

Comment:
2.2?

Reply:
@jwendell See my comment above

In maistra/istio-operator maistra-2.2 branch there is no make target 'update-2.1-charts'. and
make: *** No rule to make target 'update-2.2-charts'. Stop.
https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/38817/rehearse-38817-pull-ci-maistra-istio-operator-maistra-2.2-update-charts-in-2-3/1653843042727104512
https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/38817/rehearse-38817-pull-ci-maistra-istio-operator-maistra-2.2-update-charts-in-2-3/1653847872472354816

So remove that targets in the maistra-2.2 branch post submit jobs

Reply:
oh, nvm, I see we have  update-charts-2.2 . Let me update that